arch/arm/mach-imx/imx8m/Kconfig

```diff
@@ -2,8 +2,10 @@ if ARCH_IMX8M
 
 config IMX8M
 	bool
-	select HAS_CAAM
 	select ROM_UNIFIED_SECTIONS
+	help
+	  This architecture has CAAM but we have decided to unselect
+	  it so we can enable Foundries.io secure boot (spl + itb)
```

Comment:
No need for this extra help.

Reply:
I think there is since we removed something that should be there.

Reply:
Sure, but it is more stuff to maintain and easier to have conflicts as a consequence.

Maybe create another option that could later be upstreamed?

Reply:
but conflicts here as good. when do want changes in this area to go unnotice do we?

Reply:
I mean, it could have conflicts because of our new text, not because of the option changes. And that sort of conflict is just noise.